modules/switch/react/spec/Switch.spec.tsx

```diff
@@ -1,44 +1,123 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
+import {render, fireEvent} from '@testing-library/react';
+import canvas from '@workday/canvas-kit-react-core';
+import {ErrorType} from '@workday/canvas-kit-react-common';
 import Switch from '../lib/Switch';
 
+// Implementation details of the switch. These are meant to be helpers.
+
+// The switch track
+const getBackground = (container: HTMLElement) => {
+  return container.querySelector('div')!.querySelector('div');
+};
+
+// The switch thumb
+const getCircle = (container: HTMLElement) => {
+  return container
+    .querySelector('div')!
+    .querySelector('div')!
+    .querySelector('div');
+};
+
+const expectedBoxShadow = (c1: string, c2: string, c3: string) => {
+  return `0 0 0 2px ${c1}, 0 0 0 4px ${c2}, 0 0 0 5px ${c3}`;
+};
+
 describe('Switch', () => {
   const cb = jest.fn();
+
   afterEach(() => {
-    cb.mockReset();
+    cb.mockClear();
+  });
+
+  it('should be unchecked by default', () => {
```

Comment:
What do you think about comparing and aligning on all of these string "specs" for all the inputs so the tests are more consistent? You took a different approach [than I did in #372](https://github.com/Workday/canvas-kit/pull/372/files#diff-8e4752febc0225be3fe65f35a4b6072eR16)